Server/mods/deathmatch/logic/CMainConfig.cpp

```diff
@@ -855,44 +855,53 @@ bool CMainConfig::AddMissingSettings()
     if (!g_pGame->IsUsingMtaServerConf())
         return false;
 
-    SString strTemplateFilename = PathJoin(g_pServerInterface->GetServerModPath(), "mtaserver.conf.template");
+    const std::string templateFileName = PathJoin(g_pServerInterface->GetServerModPath(), SETTINGS_TEMPLATE_PATH);
```

Comment:
```std::string_view```?

Reply:
`std::string_view` isn't the best choice here because `PathJoin` returns a temporal object.

Reply:
Yes, but this object is no longer needed outside the function. Until the function completes and "exits", there should be a guarantee that the object created by PathJoin will exist, right?

Reply:
The lifetime of a temporary object may be extended by binding to a reference, yes. But here is not the case. The result of PathJoin will be invalidated and std::string_view will deal with a dangling pointer.